
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <title>Software Â· GitBook</title>
        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
        <meta name="description" content="">
        <meta name="generator" content="GitBook 3.2.0">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="../environment/environments.html" />
    
    
    <link rel="prev" href="../overview.md" />
    

    </head>
    <body>
        
<div class="book">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.2" data-path="../overview.md">
            
                <span>
            
                    
                    Overview
            
                </a>
            

            
        </li>
    
        <li class="chapter active" data-level="1.3" data-path="software.html">
            
                <a href="software.html">
            
                    
                    Software
            
                </a>
            

            
        </li>
    
        <li class="chapter " data-level="1.4" data-path="../environment/environments.html">
            
                <a href="../environment/environments.html">
            
                    
                    HPC Modules
            
                </a>
            

            
            <ul class="articles">
                
    
        <li class="chapter " data-level="1.4.1" data-path="../environment/gencore_annotation_1.0.html">
            
                <a href="../environment/gencore_annotation_1.0.html">
            
                    
                    Gencore_annotation_1.0
            
                </a>
            

            
        </li>
    

            </ul>
            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://www.gitbook.com" target="blank" class="gitbook-link">
            Published with GitBook
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >Software</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h1 id="software">Software</h1><h3 id="augustus">Augustus</h3><h4 id="summary">Summary</h4><p>AUGUSTUS is a gene prediction program for eukaryotes written by Mario Stanke and Oliver Keller. It can be used as an ab initio program, which means it bases its prediction purely on the sequence. AUGUSTUS may also incorporate hints on the gene structure coming from extrinsic sources such as EST, MS/MS, protein alignments and synthenic genomic alignments.</p><p><strong>Version:</strong> 3.2.2</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="bamtools">Bamtools</h3><h4 id="summary">Summary</h4><p>C++ API &amp; command-line toolkit for working with BAM data</p><p><strong>Version:</strong> 2.4.0</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="bcftools">Bcftools</h3><h4 id="summary">Summary</h4><p>Tools for dealing with BCF and VCF files</p><p><strong>Version:</strong> 1.3.1</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="bedtools">Bedtools</h3><h4 id="summary">Summary</h4><p>A powerful toolset for genome arithmetic</p><p><strong>Version:</strong> 2.25.0</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="blast">Blast</h3><h4 id="summary">Summary</h4><p>BLAST+ is a new suite of BLAST tools that utilizes the NCBI C++ Toolkit.</p><p><strong>Version:</strong> 2.2.31</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="blat">Blat</h3><h4 id="summary">Summary</h4><p>BLAT is a bioinformatics software tool which performs rapid mRNA/DNA and cross-species protein alignments.</p><p><strong>Version:</strong> 35</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="bowtie2">Bowtie2</h3><h4 id="summary">Summary</h4><p>Fast and sensitive read alignment</p><p><strong>Version:</strong> 2.2.8</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="bwa">Bwa</h3><h4 id="summary">Summary</h4><p>The BWA read mapper.</p><p><strong>Version:</strong> 0.7.15</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="circos">Circos</h3><h4 id="summary">Summary</h4><p>Circos is a software package for visualizing data and information. It visualizes data in a circular layout</p><p><strong>Version:</strong> 0.69.2</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="freebayes">Freebayes</h3><h4 id="summary">Summary</h4><p>Bayesian haplotype-based polymorphism discovery and genotyping</p><p><strong>Version:</strong> 1.0.2.29</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="genblasta">Genblasta</h3><h4 id="summary">Summary</h4><p>genBlast is a program suite, consisting of two programs: genBlastA and genBlastG. genBlastA parses local alignments, or high-scoring segment pairs (HSPs) produced by local sequence alignment programs such as BLAST and WU-BLAST and identify groups of HSPs.</p><p><strong>Version:</strong> 1.0.4</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="genblastg">Genblastg</h3><h4 id="summary">Summary</h4><p>genBlast is a program suite, consisting of two programs: genBlastA and genBlastG. genBlastA parses local alignments, or high-scoring segment pairs (HSPs) produced by local sequence alignment programs such as BLAST and WU-BLAST and identify groups of HSPs.</p><p><strong>Version:</strong> 1.38</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="hmmer">Hmmer</h3><h4 id="summary">Summary</h4><p>Biosequence analysis using profile hidden Markov models</p><p><strong>Version:</strong> 3.1b2</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="jellyfish">Jellyfish</h3><h4 id="summary">Summary</h4><p>Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA. A k-mer is a substring of length k, and counting the occurrences of all such substrings is a central step in many analyses of DNA sequence</p><p><strong>Version:</strong> 2.2.3</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="kat">Kat</h3><h4 id="summary">Summary</h4><p>KAT is a suite of tools that analyse jellyfish hashes or sequence files (fasta or fastq) using kmer counts</p><p><strong>Version:</strong> 2.0.8</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="kmergenie">Kmergenie</h3><h4 id="summary">Summary</h4><p>KmerGenie estimates the best k-mer length for genome de novo assembly</p><p><strong>Version:</strong> 1.7016</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="repeatmasker">Repeatmasker</h3><h4 id="summary">Summary</h4><p>RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.</p><p><strong>Version:</strong> 4.0.6</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="rmblast">Rmblast</h3><h4 id="summary">Summary</h4><p>RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite.</p><p><strong>Version:</strong> 2.2.28</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="samtools">Samtools</h3><h4 id="summary">Summary</h4><p>Tools for dealing with SAM, BAM and CRAM files</p><p><strong>Version:</strong> 1.3.1</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="snpeff">Snpeff</h3><h4 id="summary">Summary</h4><p>Genetic variant annotation and effect prediction toolbox</p><p><strong>Version:</strong> 4.3</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="tophat">Tophat</h3><h4 id="summary">Summary</h4><p>A spliced read mapper for RNA-Seq</p><p><strong>Version:</strong> 2.1.0</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="transdecoder">Transdecoder</h3><h4 id="summary">Summary</h4><p>TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.</p><p><strong>Version:</strong> 2.1.0</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="trinotate">Trinotate</h3><h4 id="summary">Summary</h4><p>Trinotate is a comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms</p><p><strong>Version:</strong> 3.2.2</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="vcflib">Vcflib</h3><h4 id="summary">Summary</h4><p>Command-line tools for manipulating VCF files</p><p><strong>Version:</strong> 1.0.0_rc1</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>
<h3 id="vcftools">Vcftools</h3><h4 id="summary">Summary</h4><p>A set of tools written in Perl and C++ for working with VCF files.</p><p><strong>Version:</strong> 0.1.14</p><p><strong>Conda Channel:</strong> bioconda</p><h4 id="hpc-modules">HPC Modules</h4><ul><li>gencore_annotation_1.0
</li></ul>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                <a href="../overview.md" class="navigation navigation-prev " aria-label="Previous page: Overview">
                    <i class="fa fa-angle-left"></i>
                </a>
                
                
                <a href="../environment/environments.html" class="navigation navigation-next " aria-label="Next page: HPC Modules">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"Software","level":"1.3","depth":1,"next":{"title":"HPC Modules","level":"1.4","depth":1,"path":"environment/environments.md","ref":"environment/environments.md","articles":[{"title":"Gencore_annotation_1.0","level":"1.4.1","depth":2,"path":"environment/gencore_annotation_1.0.md","ref":"environment/gencore_annotation_1.0.md","articles":[]}]},"previous":{"title":"Overview","level":"1.2","depth":1,"path":"overview.md","ref":"overview.md","articles":[]},"dir":"ltr"},"config":{"plugins":["livereload"],"root":"./_docs","styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"pluginsConfig":{"livereload":{},"highlight":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"sharing":{"facebook":true,"twitter":true,"google":false,"weibo":false,"instapaper":false,"vk":false,"all":["facebook","google","twitter","weibo","instapaper"]},"fontsettings":{"theme":"white","family":"sans","size":2},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"theme":"default","pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"variables":{},"gitbook":"*"},"file":{"path":"software/software.md","mtime":"2016-08-27T16:56:47.009Z","type":"markdown"},"gitbook":{"version":"3.2.0","time":"2016-08-27T17:02:11.102Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-livereload/plugin.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-sharing/buttons.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

