# Software

|| ||gencore_1.0 || gencore_annotation_1.0 || gencore_build_1.0 || gencore_de_novo_genomic_1.0 || gencore_de_novo_genomic_soapdenovo2_1.0 || gencore_de_novo_metagenomic_1.0 || gencore_de_novo_transcriptome_1.0 || gencore_epigenetics_1.0 || gencore_metagenomics_1.0 || gencore_perl_1.0 || gencore_python_1.0 || gencore_r_1.0 || gencore_rad_1.0 || gencore_rad_ddocent_1.0 || gencore_rnaseq_1.0 || gencore_rnaseq_1.1 || gencore_rnaseq_malaria_1.0 || gencore_variant_detection_1.0 ||

| metavelvet=1.2.02 | | | | | | 1| | | | | | | | | | | | |
| blat=35 | | 1| | 1| | 1| 1| | 1| | | | 1| | 1| 1| 1| 1|
| samtools=1.3.1 | | 1| | 1| | 1| 1| 1| 1| | | | 1| | 1| 1| 1| 1|
| stacks=1.40 | | | | | | | | | | | | | 1| | | | | |
| kat=2.0.8 | | 1| | | | | | | | | | | | | | | | |
| rmblast=2.2.28 | | 1| | | | | | | | | | | | | | | | |
| kallisto=0.43.0 | | | | | | | | | | | | | | | 1| 1| 1| |
| transdecoder=2.1.0 | | 1| | | | | | | | | | | | | | | | |
| bioconductor-deseq=1.22.1 | | | | | | | 1| | | | | | | | 1| 1| 1| |
| picard=2.0.1 | | | | | | | | | | | | | | | | | 1| |
| snpeff=4.3 | | 1| | | | | | | | | | | | | | | | 1|
| rainbow=2.0.4 | | | | | | | | | | | | | 1| | | | | |
| htseq=0.6.1.post1 | | | | | | | 1| | | | | | | | 1| 1| 1| |
| cd-hit=4.6.4 | | | | | | | | | 1| | | | 1| | | | | |
| freebayes=1.0.2.29 | | 1| | 1| | | | | | | | | 1| | | | | 1|
| bamtools=2.4.0 | | 1| | 1| | 1| 1| 1| 1| | | | 1| | 1| 1| 1| 1|
| bwa=0.7.15 | | 1| | 1| | 1| 1| 1| 1| | | | | | | | | 1|
| bedtools=2.25.0 | | 1| | 1| | 1| 1| 1| 1| | | | 1| | 1| 1| 1| 1|
| ddocent=2.1 | | | | | | | | | | | | | | 1| | | | |
| bcftools=1.3.1 | | 1| | 1| | 1| 1| 1| 1| | | | 1| | 1| 1| 1| 1|
| idba=1.1.1 | | | | 1| | | | | | | | | | | | | | |
| mira=4.0.2 | | | | 1| | | | | | | | | | | | | | |
| seqtk=1.2 | | | | 1| | 1| 1| | 1| | | | 1| | 1| 1| 1| 1|
| scipy=latest | | | | | | | | | | | 1| | | | | | | |
| abyss=1.9.0 | | | | 1| | | | | | | | | | | | | | |
| repeatmasker=4.0.6 | | 1| | | | | | | | | | | | | | | | |
| rsem=1.2.28 | | | | | | | 1| | | | | | | | | | | |
| r-ggplot2=latest | 1| | | | | | | | | | | | | | | | | |
| ipython=latest | | | | | | | | | | | 1| | | | | | | |
| genblasta=1.0.4 | | 1| | 1| | | | | | | | | | | | | | |
| blast=2.2.31 | | 1| | 1| | 1| 1| | 1| | | | 1| | | | | 1|
| python=latest | | | 1| | | | | | | | 1| | | | | | | |
| perl-dbi=latest | | | | | | | | | | 1| | | | | | | | |
| genblastg=1.38 | | 1| | 1| | | | | | | | | | | | | | |
| r-essentials=latest | 1| | | | | | | | | | | 1| | | | | | |
| bioconductor-cummerbund=2.12.1 | | | | | | | | | | | | | | | | 1| 1| |
| sickle=0.5 | | | | 1| | | 1| | | | | | | | | | | |
| gatk=3.6 | | | | 1| | | | | | | | | | | 1| 1| | 1|
| numpy=latest | | | | | | | | | | | 1| | | | | | | |
| jellyfish=2.2.3 | | 1| | 1| | | | | | | | | | | | | | |
| gatk=3.5 | | | | | | | | | | | | | | | | | 1| |
| spades=3.8.1 | | | | 1| | | | | | | | | | | | | | |
| pear=0.9.6 | | | | 1| | 1| 1| | 1| | | | | | | | | 1|
| bioconductor-edger=3.14.0 | | | | | | | 1| | | | | | | | 1| 1| 1| |
| kmergenie=1.7016 | | 1| | 1| | | | | | | | | | | | | | |
| cufflinks=2.2.1 | | | | | | | 1| | | | | | | | 1| 1| 1| 1|
| vcftools=0.1.14 | | 1| | 1| | 1| 1| 1| 1| | | | 1| | 1| 1| 1| 1|
| trimmomatic=0.36 | | | | 1| | 1| 1| | | | | | | | | 1| | |
| soapdenovo2-prepare=2.0 | | | | | 1| | | | | | | | | | | | | |
| trimmomatic=0.35 | | | | | | | | | | | | | | | | | 1| |
| hmmer=3.1b2 | | 1| | 1| | 1| 1| | 1| | | | | | | | | |
| star=2.5.2a | | | | | | | 1| | | | | | | | 1| 1| 1| 1|
| prinseq=0.20.4 | | | | 1| | 1| 1| | | | | | | | 1| 1| 1| 1|
| numpy=1.11.1 | | | | | | | | 1| | | | | | | | | | |
| bioconductor-deseq2=1.10.1 | | | | | | | 1| | | | | | | | 1| 1| 1| |
| trinotate=3.2.2 | | 1| | | | | | | | | | | | | | | | |
| python=2.7 | | | | | | | | 1| | | | | | | | | | |
| picard=2.5.0 | | | | 1| | 1| 1| | | | | | | | 1| 1| | 1|
| vcflib=1.0.0_rc1 | | 1| | 1| | 1| 1| 1| 1| | | | 1| | 1| 1| 1| 1|
| circos=0.69.2 | | 1| | 1| | 1| 1| | | | | | | | | | | 1|
| salmon=0.6.0 | | | | | | | 1| | | | | | | | | | | |
| velvet=1.2.10 | | | | 1| | | | | | | | | | | | | | |
| r=latest | | | | | | | | | | | | 1| | | | | | |
| bioconductor-biobase=latest | | | | | | | | | | | | 1| | | | | | |
| perl-termreadkey=latest | | | | | | | | | | 1| | | | | | | | |
| tophat=2.1.0 | | 1| | | | | 1| | | | | | | | 1| 1| 1| 1|
| r-base=latest | 1| | | | | | | | | | | 1| | | | | | |
| soapdenovo2=2.40 | | | | | 1| | | | | | | | | | | | | |
| discovar=52488 | | | | 1| | | | | | | | | | | | | | 1|
| bowtie2=2.2.8 | | 1| | 1| | 1| 1| 1| 1| | | | 1| | 1| 1| 1| 1|
| mawk=1.3.4 | | | | | | 1| | | 1| | | | | | | | | |
| quast=4.1 | | | | 1| | | 1| | | | | | | | | | | |
| qiime=latest | | | | | | | | | 1| | | | | | | | | |
| macs2=latest | | | | | | | | 1| | | | | | | | | | |
| augustus=3.2.2 | | 1| | 1| | 1| 1| | 1| | | | | | | | | |
| perl-app-cpanminus=latest | 1| | | | | | | | | 1| | | | | | | | |
| discovardenovo=52488 | | | | 1| | | | | | | | | | | | | | 1|
| bioconductor-biocgenerics=latest | 1| | | | | | | | | | | | | | | | | |
